<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectorFields</class>
 <widget class="QDialog" name="SelectorFields">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>245</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>405</width>
    <height>245</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>405</width>
    <height>245</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fields selector</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>81</width>
     <height>191</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTreeView" name="fieldsAval">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>192</height>
    </rect>
   </property>
   <property name="indentation">
    <number>2</number>
   </property>
  </widget>
  <widget class="QTreeView" name="fieldsUsed">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>121</width>
     <height>192</height>
    </rect>
   </property>
   <property name="indentation">
    <number>2</number>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/image/res/right.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_remove">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/image/res/left.png</normalon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectorFields</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectorFields</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
